block_additional_mappings/key_value.md

```diff
@@ -0,0 +1,17 @@
+## Key Value Pair
+
+### Key Value Pair Description
+
+The Block Additional Mapping for a `Key Value Pair` provides a generic method store data with a label in association with a Block. The label (referred to here as the `Key`) is stored within the `BlockAddIDExtraData` Element and the associate `Value` is stored within the corresponding BlockMore Element.
+
+### BlockAddIDType
+
+The BlockAddIDType value reserved for timecode is 107.
```

Comment:
Is it related to timecodes only of it could be used for anything ? If so which timecode format can use this ?

We can only have one Block Additional mapping with the `BlockAddIDType` (107) per track. Otherwise it would be impossible to tell the to which the `BlockMore` data belong to. So this format makes a loosely typed Block Addition but with a single possible use per track. And yet it doesn't tell how to interpret the data. So I'm not sure it's really useful.

Reply:
Poke on this.

Reply:
Reference to timecode is gone.